lib/email.ts

```diff
@@ -34,13 +41,16 @@ interface SendEmailParam {
   to: string
 }
 
+const textRenderer = new MarkdownTextRenderer()
 export const sendEmails = async (params: SendEmailParam[]): Promise<void> => {
   const { mdTemplate, subjectTemplate } = useTemplateStore.getState()
   return Promise.all(
     params.map(({ user, to }) => {
       const subject = renderTemplate(subjectTemplate, user)
-      const body = md.render(renderTemplate(mdTemplate, user))
-      return _sendEmail({ to, subject, body })
+      const md = renderTemplate(mdTemplate, user)
+      const html = marked(md, { async: false }) as string
+      const text = marked(md, { renderer: textRenderer, async: false }) as string
```

Comment:
> For this, instead of using `marked`, you can just use `const text = md` directly. Does that simplify things? Can we avoid the patch and use the old markdown library?

#### marked
1. [`marked.js`](https://github.com/markedjs/marked) supports plain text rendering more easily (by using custom renderer), so I used it.  I thought it's better to show plain text than showing markdown specific symbols.
 This was the simplest option that I could do

#### `mimetext`
1. This module reduces the boilerplate of generating raw email contents for both text/html
2. `@types/mimetext` exists, but the type definitions are mismatching and causes runtime error.
3. The patch is to add type exports to the module. There are related PRs but not merged yet.

https://github.com/muratgozel/MIMEText/pull/58
https://github.com/muratgozel/MIMEText/pull/50

Reply:
OK -- as long as you're happy with it.